examples/Azure_IoT_Central_ESP32/iot_configs.h

```diff
@@ -5,10 +5,54 @@
 #define IOT_CONFIG_WIFI_SSID              "SSID"
 #define IOT_CONFIG_WIFI_PASSWORD          "PWD"
 
+// Define if using device x509 certificates
+// #define IOT_CONFIG_USE_X509_CERT
+
+#ifdef IOT_CONFIG_USE_X509_CERT
+
+/*
+
+Device certificate should be of the form:
```

Comment:
1. I don't understand this comment. Is this supposed to be the value of IOT_CONFIG_DEVICE_CERT ?
i.e. replace "Device Certificate" with a string containing the PEM format?

2. Do we really need the `\r\n` or will `\n` be enough?

Reply:
Updated the comment. Maybe the \n but seems to be working here.